consent/strategy_default.go

```diff
@@ -696,13 +695,9 @@ func (s *DefaultStrategy) executeBackChannelLogout(ctx context.Context, r *http.
 		tasks = append(tasks, task{url: c.BackChannelLogoutURI, clientID: c.OutfacingID, token: t})
 	}
 
-	var wg sync.WaitGroup
 	hc := httpx.NewResilientClient()
-	wg.Add(len(tasks))
 
 	var execute = func(t task) {
-		defer wg.Done()
-
 		res, err := hc.PostForm(t.url, url.Values{"logout_token": {t.token}})
```

Comment:
Retry could take 5 x 60sec request timeout + backoff time in between. We could generate updated logout token for each request so `iat` does not expire.
```golang
var body = retryablehttp.ReaderFunc(func() (io.Reader, error) {			
	return strings.NewReader(url.Values{"logout_token": {generateToken}}.Encode()), nil
})
req, err := retryablehttp.NewRequest("POST", c.BackChannelLogoutURI, body)
if err != nil {
	s.r.Logger().WithRequest(r).WithError(err).
		WithField("client_id", c.OutfacingID).
		WithField("backchannel_logout_url", c.BackChannelLogoutURI).
		Error("Unable to create OpenID Connect Back-Channel Logout Request")
	return
}
req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
res, err := hc.Do(req)
if err != nil {
	s.r.Logger().WithRequest(r).WithError(err).
		WithField("client_id", c.OutfacingID).
		WithField("backchannel_logout_url", c.BackChannelLogoutURI).
		Error("Unable to execute OpenID Connect Back-Channel Logout Request")
	return
}
```